### ISSUE TITLE
Fails to read manifest with a thumbnail

### ISSUE THREAD

**Issue by rlskoeser** [iiif-prezi/iiif-prezi#25]
Reported error is  `Every resource must have @type`

**Comment by zimeon** [comment]
@rlskoeser - could you post the manifest?

**Comment by rlskoeser** [comment]
@zimeon sure, here's one:
https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest

Initializing the manifest reader based on what I read in the brief usage doc:
```
import requests
from iiif_prezi.loader import ManifestReader

response = requests.get('https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest')
reader = ManifestReader(response.json())
manifest = reader.read()
```

I get this error:
```
  File "/Users/rsk/workarea/env/iiif/lib/python3.5/site-packages/iiif_prezi/loader.py", line 148, in read
    top = self.readObject(js)
  File "/Users/rsk/workarea/env/iiif/lib/python3.5/site-packages/iiif_prezi/loader.py", line 305, in readObject
    subo = self.readObject(v, what, k)
  File "/Users/rsk/workarea/env/iiif/lib/python3.5/site-packages/iiif_prezi/loader.py", line 185, in readObject
    raise RequirementError('Every resource must have @type', parent)
iiif_prezi.factory.RequirementError: Every resource must have @type
```

**Comment by zimeon** [comment]
Verified with 0.2.7:

```
(py3)simeon@RottenApple iiif-prezi (master)> python check_remote_manifests.py https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest
------
https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest
   => RequirementError: Every resource must have @type
```

**Comment by azaroth42** [comment]
Fixed (again) in 0.2.8. Not sure how we lost this fix.

**Comment by zimeon** [comment]
Still fixed in 0.2.9, not sure why we didn't close before:

```
simeon@Cider iiif-prezi (master %)> python check_remote_manifests.py https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest
------
https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest
```
